components/Tabs2/tests/snapshot/MDCTabBarViewItemViewSnapshotTests.m

```diff
@@ -27,9 +27,6 @@ @implementation MDCTabBarViewItemViewSnapshotTests
 - (void)setUp {
   [super setUp];
 
-  // Uncomment below to recreate all the goldens (or add the following line to the specific
```

Comment:
Please do not remove this comment. We leave it in-place as a convention and also to make it easier to recreate the goldens.